pages/common/twine.md

```diff
@@ -0,0 +1,21 @@
+# twine
+
+> Twine is a utility for publishing Python packages on PyPI.
+> Subcommands such as `twine upload` have their own usage documentation.
+> More information: <https://twine.readthedocs.io>.
+
+- Upload to the Test PyPI [r]epository and verify things look right:
+
+`twine upload -r testpypi dist/*`
+
+- Upload to PyPI:
```

Comment:
I would put the Upload to PyPI first, before test upload.

The upload itself is after all the main usage, the test upload is optional.